src/cryptominisat.h.in

```diff
@@ -132,8 +132,14 @@ namespace CMSat {
         ////////////////////////////
 
         uint64_t get_sum_conflicts(); //get total number of conflicts of all time of all threads
+        //! Return sum of all conflicts since construction across all the threads
+        uint64_t get_sum_conflicts() const;
         uint64_t get_sum_propagations();  //get total number of propagations of all time made by all threads
+        //! Returns sum of all propagations since construction across all the threads
+        uint64_t get_sum_propagations() const;
         uint64_t get_sum_decisions(); //get total number of decisions of all time made by all threads
+        //! Returns sum of all decisions since construction across all the threads
+        uint64_t get_sum_decisions() const;
 
```

Comment:
Please use the setup I have -- having the comments after the function. I understand that's not how others do it, but either we have it one way or the other, we can't have it both ways. Thanks!